ports/espressif/boards/Lilygo-T-oi-Plus-ESP32C3/mpconfigboard.mk

```
@@ -0,0 +1,10 @@
+CIRCUITPY_CREATOR_ID = 0xC3C30000
+CIRCUITPY_CREATION_ID = 0x00C30001
+
+IDF_TARGET = esp32c3
+
+INTERNAL_FLASH_FILESYSTEM = 1
+
+CIRCUITPY_ESP_FLASH_MODE = dio
+CIRCUITPY_ESP_FLASH_FREQ = 80m
+CIRCUITPY_ESP_FLASH_SIZE = 4MB
```

```suggestion
CIRCUITPY_ESP_FLASH_SIZE = 4MB

```
Add newline at the end of file.